File: modul_4/PRAK405-2510817220008-NamiraHumaira.py
baris, kelipatan = map(int, input().split())
total = 0
for i in range(1, baris + 1, 1):
    jumlah = 0
    print("(", end="")
    for j in range(i, 0, -1): 
        jumlah += j * kelipatan
        print(f"{j} * {kelipatan}" , end="")
        if j > 1: print(") + (", end="")
    print(f") = {jumlah}")
    total += jumlah
print(f"{total}")

File: modul_4/PRAK405-2510817220008-NamiraHumaira.c
#include <stdio.h>
int main() 
{
    int kelipatan, baris;
    int total = 0;
    scanf("%d %d", &baris, &kelipatan);

    for(int i = 1; i <= baris; i++)
    { 
        int jumlah = 0; 
        printf("(");
        for(int j = i; j >= 1; j--)
        {
            jumlah += j * kelipatan;
            printf("%d * %d", j , kelipatan);
            if(j > 1) printf(") + (");
        }
        printf(") = %d \n", jumlah);
        total += jumlah;
    }
    printf("%d \n", total);
    return 0;
}